test/routes/graphql/Query/organizations.test.ts

```
@@ -0,0 +1,153 @@
+			const result = await mercuriusClient.query(Query_organizations, {
+				variables: {
+					input: {
+						id: knownOrganizationId,
+					},
+				},
+			});
+
+			expect(result.errors).toBeUndefined();
+			expect(result.data.organizations).toHaveLength(1);
+			expect(result.data.organizations).not.toBeNull();
+			expect(result.data.organizations![0]).toEqual(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)